frontend/src/styles.scss

```
@@ -31,25 +31,29 @@ html {
     &:hover {
-      background: rgb(128 128 128 / 70%);
+      background: var(--surface-500);
     }
   }
 
   scrollbar-width: thin;
-  scrollbar-color: rgb(128 128 128 / 50%) rgb(0 0 0 / 10%);
+  scrollbar-color: var(--surface-600) transparent;
+}
+
+::ng-deep .p-progressbar .p-progressbar-value {
```

Stylelint (17.9.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)